prod/westeurope/eucovidcert/functions_eucovidcert/function_app_slot_staging/terragrunt.hcl

```diff
@@ -0,0 +1,89 @@
+dependency "resource_group" {
+  config_path = "../../resource_group"
+}
+
+dependency "subnet" {
+  config_path = "../subnet"
+}
+
+# Common
+dependency "application_insights" {
+  config_path = "../../../common/application_insights"
+}
+
+dependency "key_vault" {
+  config_path = "../../../common/key_vault"
+}
+
```

Comment:
missing azure devops subnet

Reply:
Done in [33ca10c](https://github.com/pagopa/io-infrastructure-live-new/pull/529/commits/33ca10ce7b6c70fb338f34d180f670d1c3d7a30f) only in _staging_